.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,19 @@
+name: unit tests
```

Comment:
Nitpick: You capitalized all other workflow names :)

Reply:
Done